jobs/influxdb/templates/bin/influxdb_setup.erb

```diff
@@ -72,11 +72,11 @@ ADMIN_USERNAME="<%= admin_username %>"
 ADMIN_PASSWORD_FILE="/var/vcap/store/influxdb/admin_password_file"
 if [ ! -f "$ADMIN_PASSWORD_FILE" ]; then
   # generate the admin password and store it in /var/vcap/store
-  openssl rand -hex 16 > "$ADMIN_PASSWORD_FILE"
+  xxd -l 16 -pu </dev/urandom >"$ADMIN_PASSWORD_FILE"
```

Comment:
looks likes `u` here is useless:

```
$ xxd -l 16 -pu /dev/urandom
2770fa33e13d46973237ce69ea5beaa1
```

probably it was supposed to be:

```
$ xxd -u -l 16 -p /dev/urandom
C87E178B1E4448EF0053023A106F83D2
```

or without `u`